model/jpa/src/main/resources/META-INF/orm.xml

```diff
@@ -48,7 +48,8 @@ http://java.sun.com/xml/ns/persistence/orm_2_0.xsd"
             </many-to-one>
             <many-to-many name="categories" fetch="EAGER">
                 <cascade>
-                    <cascade-all/>
+                    <cascade-persist/>
+                    <cascade-merge/>
                 </cascade>
```

Comment:
perfect! should that particular change be migrated to 1.0.x ?

On Fri, Jan 23, 2015 at 9:43 AM, Erik Jan de Wit <notifications@github.com>
wrote:

> In model/jpa/src/main/resources/META-INF/orm.xml
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/473#discussion_r23437540>
> :
>
> > @@ -48,7 +48,8 @@ http://java.sun.com/xml/ns/persistence/orm_2_0.xsd"
> >              </many-to-one>
> >              <many-to-many name="categories" fetch="EAGER">
> >                  <cascade>
> > -                    <cascade-all/>
> > +                    <cascade-persist/>
> > +                    <cascade-merge/>
> >                  </cascade>
>
> You can't remove the categories that still have relations, this failed
> with the improved tests
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/473/files#r23437540>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf